src/analyses/apron/apronAnalysis.apron.ml

```diff
@@ -104,14 +104,17 @@ struct
       {st with apr = apr'}
     )
 
-  let assign_to_global_wrapper ask getg sideg st lv f =
+  let rec assign_to_global_wrapper (ask:Queries.ask) getg sideg st lv f =
     match lv with
     (* Lvals which are numbers, have no offset and their address wasn't taken *)
     (* This means that variables of which multiple copies may be reachable via pointers are also also excluded (they have their address taken) *)
     (* and no special handling for them is required (https://github.com/goblint/analyzer/pull/310) *)
```

Comment:
I'm having hard time understanding what this comment is trying to say: "Lvals which are _numbers_", what?
The second half seems to be about `IsMultiple`, but now we're accounting for it, so that's no longer true, I suppose.

Reply:
I removed the comment completely now, it was outdated.